jenkins/bootstrap.py

```diff
@@ -735,11 +737,16 @@ def job_args(args):
     return [os.path.expandvars(a) for a in args]
 
 
-def job_script(job, use_json):
+def job_script(job, use_json, jobs_dir):
     """Return path to script for job."""
+    jobs_dir_resolver = lambda f: test_infra('jobs/%s' % f)
+
+    if jobs_dir:
+        jobs_dir_resolver = lambda f: os.path.join(jobs_dir, f)
+
     if not use_json:
```

Comment:
ref https://github.com/kubernetes/test-infra/issues/3711 and https://github.com/kubernetes/test-infra/pull/3825

How is your bootstrap.py get updated? Sync with here every so often? I would probably port over them to use [the simple execute scenario](https://github.com/kubernetes/test-infra/blob/master/scenarios/execute.py) with a call into the script, like [this job](https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L13-L21) maybe.